hackathon_site/registration/jinja2/registration/application.html

```diff
@@ -97,7 +97,7 @@
 
     select2.data('select2').$selection.css('height', '2.4375rem');
 
-    {% if form.school.value %}
+    {% if form.school.value() and form.school.value() is not none %}
```

Comment:
Isn't this redundant cause if form.school.value() is something it won't be none

If anything you'd wanna be testing for the not none first then the form.school.value()

Reply:
From testing, it seems that in jinja that `None` comes out to be truthy. Without that extra bit, if you manage to submit the form with empty fields (which you have to remove all the `required` attributes to do), they will come back with the string "None" as the value.

Reply:
Got it, would still be best practice to test for none first